src/main/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/configuration/KafkaConfiguration.java

```diff
@@ -73,7 +76,7 @@ ConsumerConfig getRetryConsumerConfig() {
         ConsumerConfig config = new ConsumerConfig();
         config.setBrokerAddresses(new String[]{brokerAddress});
         config.setTopics(Collections.singletonList(retryTopicName));
-        config.setPollTimeout(pollTimeout);
+        config.setPollTimeout(retryThrottleSeconds * 2 * 1000);
```

Comment:
just curious what the thought process is by doubling the value?
And if theres any possible issues with particuarly high timeouts?

Reply:
It was the first thing I thought of really, if it hasn't polled iby the second one then it is probably dead.

for the timeouts, look slike there is no max (see below it has [1,...])

https://docs.confluent.io/platform/current/installation/configuration/consumer-configs.html#max.poll.interval.ms